scheduler/scheduler-smartproxy-common/src/main/java/org/ow2/proactive/scheduler/smartproxy/common/AbstractSmartProxy.java

```diff
@@ -800,9 +836,12 @@ protected void updateJob(NotificationData<?> notification) {
             }
             case FAILED: {
                 log.debug("The job " + id + " is failed.");
-                // removeAwaitedJob(id.toString());
+                jobTracker.removeAwaitedJob(id.toString());
                 break;
             }
+            default: {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Extract this nested code block into a method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1199)